src/components/PublicToggle/PublicToggle.js

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import cx from 'classnames'
+import SwitchStyled from 'components/SwitchStyled'
+import './PublicToggle.scss'
+const { func, bool } = PropTypes
+
+export default class PublicToggle extends Component {
+  static propTypes = {
+    isPublic: bool,
+    togglePublic: func
+  }
+
+  static defaultProps = {
+    isPublic: false
+  }
+
+  render () {
+    const { isPublic, togglePublic } = this.props
+
+    return (
+      <div styleName={cx('publicToggleContainer', { postIsPublic: isPublic })}>
+        <div styleName='publicToggle'>
+          <SwitchStyled checked={isPublic} onChange={togglePublic} backgroundColor={isPublic ? '#0DC39F' : '#8B96A4'} />
```

Comment:
nice, using the new component Tom made :)

Reply:
yeah it works great!